app/main.py

```diff
@@ -1 +1,52 @@
-# write your code here
+import os
+
+
+def check_if_mv(command: str) -> None:
+    if command != "mv":
+        raise ValueError("Command must begging with 'mv' without flags")
+
+
+def create_dirs_from_str(destination: str) -> None:
+    if "/" in destination:
+        path = os.path.split(destination)[0]
+        if not os.path.exists(path):
+            os.makedirs(path)
+
+
+def relocate_file_if_path_exists(source_file: str, destination: str) -> None:
+    if os.path.isdir(destination):
+        if "/" in source_file:
+            source_file_name = os.path.split(source_file)[1]
+            os.rename(source_file, f"{destination}{source_file_name}")
+        else:
+            os.rename(source_file, f"{destination}{source_file}")
+
+
+def rewrite_existed_file(source_file: str, destination: str) -> None:
+    if os.path.isfile(destination):
+        user_input = input(
+            f"Do you want rewrite {destination} with {source_file}: yes/no?"
+        )
+        if user_input.lower() == "no":
+            raise ValueError("Please, rename destination file")
+
+
+def move_file(command: str) -> None:
+    try:
+        mv_command, source_file, destination = command.split()
+    except ValueError:
```

Comment:
except ValueError and print() than fail.
 Use raise ValueError(your message) instead of print

Reply:
But [a bit earlier, Kate wrote](https://github.com/mate-academy/py-move-file/pull/177#discussion_r1092419721), that's bad practice.
Which way is better?